### DISCUSSION TITLE
Redistributable TMS based package tool like osgearth_package

### DISCUSSION THREAD

**Discussion by stevegbarry** [pelicanmapping/osgearth#2131] [Support Q&A] [answered]
In older versions of osgEarth (prior to 3.0) there was a tool called osgearth_package that created a TMS repo.  It looks as if the tool is no longer supported (since 3.0).  Is there another tool available to create a TMS repo?

**Comment by jasonbeverage** [comment, marked as answer]
You can use the new osgearth_conv tool to do the same thing:  http://docs.osgearth.org/en/latest/data.html#building-tilesets

**Reply by stevegbarry** [reply]
Thanks.
